AIPscan/API/namespace_data.py

```diff
@@ -22,19 +29,12 @@
 """
 
 
-def parse_bool(val, default=True):
-    try:
-        return bool(strtobool(val))
-    except (ValueError, AttributeError):
-        return default
-
-
-@api.route("/aip-overview/<storage_service_id>")
+@api.route("/storage_service/<storage_service_id>/aip-overview")
```

Comment:
Thanks for this Ross - this is a really nice and helpful backstory/rationale and I appreciate the detail! Your thinking makes sense to me too.

Let me start by saying that I admit I may have overreached here, so very possibly the best thing to do is to simply revert the routes to how they were before.

My motivation for this commit was always inspired a bit by the idea of cool URIs, and particularly how I typically expect the components of the URI to be in a semantically meaningful order, especially when some sort of identifier is being used.

So looking at the `/api/ingest/<SIP UUID>/delete/` Archivematica API endpoint as an example, we might see something in the wild like: `https://example.com/api/ingest/9259c0bb-5636-4420-a6f3-ee283b32d5a4/delete`. The UUID here relates for the semantic unit that immediately precedes it (in this case "ingest"), and the next component of the URI (in this case "delete"), is an action taken on the resource previously identified.

An existing AIPscan API endpoint `fmt-overview/<storage_service_id>` might look in the wild like this: `https://example.com/api/fmt-overview/1`. I found it odd that the ID here is identifying a resource type that's not found in the URI, hence the reordering.

> If I /try/ to read your intention - I wonder if perhaps between the two positions we'll find that storage service ID might be better as a query parameter? IDK.

You know, I think you might be right here! But maybe again these are higher-level questions that would benefit from a more direct focus on the API rather than done as a "by the way" commit in a PR like this, so I will revert them for now.

Reply:
And on a higher-level note, I'm recognizing here that on my end there are probably two issues with how I approached this: 1. trying to cram it in a only-sort-of-related PR, and 2. not communicating my intentions clearly enough, either in code, an issue, or elsewhere. So room for improvement on my end, which is noted and I will try to bear in mind moving forward!